frontend/public/components/utils/headings.tsx

```diff
@@ -50,7 +50,7 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
 
   const logo = isCSV
     ? csvLogo()
-    : <div className="co-m-pane__name co-resource-item">{ kind && <ResourceIcon kind={kind} className="co-m-resource-icon--lg" /> } <span id="resource-title" className="co-resource-item__resource-name">{resourceTitle}</span></div>;
+    : <div className="co-m-pane__name co-resource-item">{ kind && <ResourceIcon kind={kind} className="co-m-resource-icon--lg" /> } <span id="resource-title" className="co-resource-item__resource-name" data-test-id="resource-title">{resourceTitle}</span></div>;
```

Comment:
I'd remove `id` and update the existing tests that use `#resource-title` to use the `data-test-id`